minio/minioadmin.py

```diff
@@ -322,6 +323,14 @@ def info(self) -> str:
         )
         return response.data.decode()
 
+    def account_info(self) -> str:
```

Comment:
Refer https://github.com/minio/madmin-go/blob/main/user-commands.go#L80
1. Add comment as per `madmin-go`
2. Add argument `prefix_usage: bool = false` and handle it

Reply:
Fixed.